src/desktop/apps/artwork/components/commercial/view.coffee

```diff
@@ -40,9 +40,19 @@ module.exports = class ArtworkCommercialView extends Backbone.View
   acquire: (e) ->
     e.preventDefault()
 
-    # Show the new buy now flow if you have the lab feature enabled
     loggedInUser = CurrentUser.orNull()
-    if loggedInUser?.hasLabFeature('New Buy Now Flow')
+    if artwork.partner.type == "Auction" or artwork.partner.type == "Auction House"
```

Comment:
This is a #minor thing, but if we check for the presence of the lab feature first, then as admins we'll be able to test the new flow in all scenarios before we launch. I think here and in the mobile case it's okay to _always_ use the new flow if you have the lab feature enabled.